pysindy/pysindy.py

```diff
@@ -351,7 +351,7 @@ def equations(self, precision=3):
             precision=precision,
         )
 
-    def print(self, lhs=None, precision=3):
+    def print(self, lhs=None, precision=3, flush=False):
```

Comment:
```suggestion
    def print(self, lhs=None, precision=3, *, flush=False):
```

To shadow use in the builtin `print()`, let's require that `flush` be passed as a keyword argument.